thirdparty_containers/system-test/playlist.xml

```diff
@@ -0,0 +1,273 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TestConfig/playlist.xsd">
+	 <test>
+                <testCaseName>MauveSingleThreadLoadTest</testCaseName>
```

Comment:
Please do not list all targets as we actively add/remove/modify system test targets.
Please do it the same as your functional test PR, just run `sanity.system`.

Reply:
@llxia 

Our plan was to run only some of the mauve and shared classes tests in containers. Not the entire sanity.system. Shared classes tests are listed as extended.system, so if we go by these groups we would need to run extended.system.

Reply:
Those were the initial set of tests that we wanted to run in containers.  We just wanted some stress test and check shared classes related test to check the JVM optimizations included for containers.